src/js/widget.radioboxes.js

```diff
@@ -50,7 +50,7 @@
                 input.data('value', value);
             }
             if (selected) {
-                input.prop('checked', true);
+                input.attr('checked', true);
```

Comment:
To dla mnie trochę zaskoczenie. Dlaczego prop przestało działać? Wydawało mi się, że to jest jedyna poprawna metoda obsługi "checked" via jquery. Zobacz np. tutaj: https://stackoverflow.com/a/24595458

Nie mówię, że ta poprawka jest koniecznie zła, ale jest to trochę podejrzane. Na ile jesteś pewien, że jest poprawna?